app/controllers/public_controller.rb

```diff
@@ -138,6 +109,13 @@ def contact_us
 
   private
 
+  def validate_order
+    valid_orders = %w[score name stake random]
```

Comment:
Maybe create a constant in the model?

Reply:
Nope - it's different for API, public, and validator controllers.